src/util/createServerRootMixin.js

```diff
@@ -132,13 +132,16 @@ function augmentInstantSearch(
    */
   search.findResultsState = function(componentInstance) {
     let app;
+    let clonedComponentInstance;
 
     return Promise.resolve()
       .then(() => {
         app = cloneComponent(componentInstance, {
           mixins: [
             {
               created() {
+                // eslint-disable-next-line consistent-this
+                clonedComponentInstance = this;
```

Comment:
looks like you reverted this?

Reply:
yeah it turned out there was a reason why I did that in the first place, which I forgot about.

```js
app = createApp({
  created() {
    console.log(this);
  },
  methods: {
    hello() { }
  }
})
```

In vue 3, that `app` is an app instance which includes methods regarding apps like `mount, use, provide, ...`.

And `this` inside `created()` is an actual component (root component in this case) instance that includes `data` and the methods like `hello`.